queue_job/data/queue_data.xml

```diff
@@ -8,7 +8,9 @@
             <field name="numbercall">-1</field>
             <field ref="model_queue_job" name="model_id" />
             <field name="state">code</field>
-            <field name="code">model.requeue_stuck_jobs()</field>
+            <field
+                name="code"
+            >model.requeue_stuck_jobs(enqueued_delta=5, started_delta=0)</field>
```

Comment:
thought: what if we used a default value of the server's limit_time_real + 60s? 🤔

Reply:
Maybe use False if you want to use config settings + 1min and 0 if you want to disable it?

Reply:
Given that False and 0 are both falsy, instead -1 for disable?

Reply:
I'd say 0 for disable (to keep compatibility with current behavior) and -1 to auto-configure.